### PR TITLE
portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by solog0039**
In this project
1. githu workflow created
2. CSS and HTML for toolbar and headline sections completed
3. tested

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>protflio main page</title>
+</head>
+<body >
+
+  <section class="toolbar">
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change this from `section` to header tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <div class="name"> 
+     <h1> Solomon</h1>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please change the `h1` tag to `a` tag as the logo must be in an `a` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+</head>
+<body >
+
+  <section class="toolbar">
+    <div class="name"> 
+     <h1> Solomon</h1> 
+    </div>
+    <div class="Menu">
+     <img src="toolbar-images/Union.png" alt="MENU">
+    </div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please wrap the hamburger button inside a `nav` tag.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background: #1c1a19;
+}
```
Comment (ShahierNashaat) [inline style.css]: - [ ] Please make the `section` and the `header` responsive with the mobile size as they have to be in the Figma design suggestions you can remove display: absolute and make the width of the `header` and the `section` 100%.

your design 
Screenshot from 2022-04-28 00-32-48

Figma design
Screenshot from 2022-04-27 20-47-24

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @solog0039 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by solog0039** [comment]
changes requested by reviewer @ShahierNashaat  has been made. Project ready to ask for new review.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```
Comment (Munsa1) [inline index.html:1]: Great work implementing most of the requested changes.
- [ ] As stated by the previous reviewer, Kindly make your design responsive, you will need to do so in order to have a professional-looking portfolio. 
Your design 
image

Figma Design
image

- The issue is that as you develop your site you are not using the inspect tool to see how your site looks like in mobile view. [Check out this video](https://www.youtube.com/watch?v=x8ofsJiELQ0)
- Once you have opened your site in mobile view use CSS to style your portfolio to match the Figma template you chose.
- Width properties and height properties should be used with caution they make your site to be unresponsive. For example, if you give the `header` section width of 300px, this means when viewing your site on mobile as the screen gets bigger than 300px the header will become static and white space will remain to cause the issue that you're having right now. I would advise removing all width and height properties or making all of them 100% as the previous reviewer advised. [See this article](https://www.w3schools.com/html/html_responsive.asp)

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @solog0039  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good PR :heavy_check_mark: 
- Linters Passing :heavy_check_mark: 
- Readme file :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+       <div class="heading-1">
+            <h1 class="heading-1 margin-24">Hi there,</h1>
+            <h1 class="heading-1 margin-24">I'm Solomon</h1>
+       </div>
+       <h2 class="heading-2 text-color margin-24">I'm a software Developer</h2>
+       <p class="section-par text-color margin-24">I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+    see and have a project you need coded,
+    don’t hestiate to contact me..</p>
+       <div class="social-media">
+            <img  class="github" src="socialmedia/github.png" alt="github">
```
Comment (danielufeli) [inline index.html:35]: - [Optional] Kindly link each social icon using (`a` tag) for better user interaction.

**Review by danielufeli — APPROVED** [review]
Hi @solog0039,

Great work on making the changes requested by the previous reviewer 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**